package/src/utils/function/isNumber/isNUmber.ts

```
@@ -0,0 +1,3 @@
+export function isNumber(value: any): boolean {
+    return typeof value === 'number' && !isNaN(value);
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)